test/integration/scale/scale_test.go

```diff
@@ -55,18 +55,31 @@ func TestMain(m *testing.M) {
 func TestScaleSubresources(t *testing.T) {
 	clientSet, tearDown := setupWithOptions(t, nil, []string{
 		"--runtime-config",
-		"api/all=true",
+		// TODO(liggitt): remove these once apps/v1beta1, apps/v1beta2, and extensions/v1beta1 can no longer be served
+		"api/all=true,extensions/v1beta1/deployments=true,extensions/v1beta1/replicationcontrollers=true,extensions/v1beta1/replicasets=true",
 	})
 	defer tearDown()
 
-	_, resourceLists, err := clientSet.Discovery().ServerGroupsAndResources()
+	resourceLists, err := clientSet.Discovery().ServerResources()
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	expectedScaleSubresources := map[schema.GroupVersionResource]schema.GroupVersionKind{
 		makeGVR("", "v1", "replicationcontrollers/scale"): makeGVK("autoscaling", "v1", "Scale"),
 
+		// TODO(liggitt): remove these once apps/v1beta1, apps/v1beta2, and extensions/v1beta1 can no longer be served
```

Comment:
Looks like those code are in commit "Updated integration test cases" but not in overall changes. How is this happened? Possibly merged other's code changes? Please use rebase instead of merge

Reply:
It might be introduced by multiple rebases and fetch different branches. It is outdated means it has already cleared before being reviewed.